extras/mmu_machine.py

```diff
@@ -458,6 +466,30 @@ def select_gear_stepper(self, gate):
         else:
             self._reconfigure_rail(None)
 
+    def _register(self, toolhead, stepper, trapq = None):
+        # klipper 0.13.0 <= 195
+        # if hasattr(toolhead, 'register_step_generator'):
+        #     if stepper.generate_steps not in toolhead.step_generators:
+        #         toolhead.register_step_generator(stepper.generate_steps)
```

Comment:
I wonder if we still need to fiddle with the step generators on older versions of klipper or if changing the trapq is sufficient?